lib/kennel/utils.rb

```diff
@@ -5,6 +5,7 @@ module Utils
 
     class TeeIO < IO
       def initialize(ios)
+        super(0)
```

Comment:
That's an unfamiliar construct and worth a code comment, I think. Why 0?

Reply:
ah yeah ... a bogus value to make it stop complaining :D
... I'll add a comment

Reply:
updated